readinglist/backend/__init__.py

```diff
@@ -36,6 +36,16 @@ def get_all(self, resource, user_id, filters=None, sorting=None,
                 pagination_rules=None, limit=None):
         raise NotImplementedError
 
+    def strip_deleted_record(self, resource, user_id, record):
+        """Strip the record to its deleted version (in place)."""
```

Comment:
I would rather use a deleted_items hash table per user with ``{"id": <deletion_last_modified>}`` to avoid having deleted items mixed with used ones.